packages/mcfunction/src/node/command.ts

```diff
@@ -26,7 +26,7 @@ export interface CommandMacroNode extends core.AstNode {
 export const CommandMacroNode = Object.freeze({
 	is<T extends core.DeepReadonly<core.AstNode> | undefined>(
 		obj: T,
-	): obj is core.NodeIsHelper<CommandMacroNode, T> {
+	): obj is core.PotentiallyReadonly<CommandMacroNode, T> {
```

Comment:
When should we use namespaces and when `Object.freeze`? What is the difference?

Reply:
I only stopped using `namespace` since it's a TS-specific thing instead of a JS thing. We should decide on a style and keep using that.